misc/dist-test-cases

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "EXTRA_DIST = misc/units \\" > "$1"
+find Tmain Units -type f | sed -e 's/$/\\/' >> "$1"
+echo '$(NULL)' >> "$1"
```

Comment:
BTW, finding all files is a bad idea, if e.g. there are leftover *.tmp* files in the test trees.